Functions/CatalogItems/Remove-RsCatalogItem.ps1

```diff
@@ -33,10 +50,11 @@ function Remove-RsCatalogItem
         
         $Proxy,
         
-        [Parameter(Mandatory=$True)]
+        [Parameter(Mandatory=$True,ValueFromPipeline = $true,ValueFromPipelinebyPropertyname = $true)]
         [string]
         $Path
     )
+ process {
```

Comment:
also please move the bracket to the next line (to keep it consistent with our style in the code base)